mutfunc.pm

```diff
@@ -0,0 +1,477 @@
+=head1 LICENSE
+
+Copyright [1999-2015] Wellcome Trust Sanger Institute and the EMBL-European Bioinformatics Institute
+Copyright [2016-2022] EMBL-European Bioinformatics Institute
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+=head1 CONTACT
+
+ Ensembl <http://www.ensembl.org/info/about/contact/index.html>
+    
+=cut
+
+=head1 NAME
+
+ mutfunc
+
+=head1 SYNOPSIS
+
+ mv mutfunc.pm ~/.vep/Plugins
+ ./vep -i variations.vcf --plugin mutfunc,motif=1,db=/FULL_PATH_TO/mutfunc_human_data.db
+ ./vep -i variations.vcf --plugin mutfunc,all=1,file=/FULL_PATH_TO/tfbs_tab.gz,db=/FULL_PATH_TO/mutfunc_human_data.db
+
+=head1 DESCRIPTION
+
+ A VEP plugin that retrieves data from mutfunc db predicting destabilization of protein structure, interaction. regulatory region etc.
+ 
+ Please cite the IntAct publication alongside the VEP if you use this resource:
+ https://www.embopress.org/doi/full/10.15252/msb.20188430
+ 
+ Pre-requisites:
+ 
+ 1) mutfunc tfbs file can be downloaded from -
+ http://ftp.ebi.ac.uk/pub/databases/mutfunc/mutfunc_v2/ 
+
+ mutfunc db can be downloaded from - ???
```

Comment:
The location of the database file needs to be updated when the location is decided.